app/components/ui/molecules/SearchBox.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import { escapeRegExp } from 'lodash'
+import theme from '@elifesciences/elife-theme'
+
+import SearchBox from './SearchBox'
+
+const searchOptions = [
+  { value: 'first option' },
+  { value: 'final option' },
+  { value: 'second option' },
+]
+
+function filterFunction(options, searchValue, field) {
+  if (!searchValue) return options
+
+  const inputValue = searchValue.trim().toLowerCase()
+  if (!inputValue) return options
+
+  return options.filter(option =>
+    option[field].toLowerCase().includes(inputValue),
+  )
+}
+
+function getMatchIndex(inputValue, option) {
+  const re = new RegExp(escapeRegExp(inputValue))
+  const match = re.exec(option.toLowerCase())
+  if (match) return match.index
+  return -1
+}
+
+function makeWrapper() {
+  return mount(
```

Comment:
There's no theme prop being passed in to `Autosuggest` any more, so I reckon you can just do a shallow render here, as I don't think you need the `ThemeProvider`?

Reply:
The `theme` is used in the `AutosuggestWrapper`, which is part of the `SearchBox`. I didn't spend that much time digging up in enzyme to try to figure out a weird way to make it work with `shallow` (since it doesn't, e.g. there's no straightforward way to reach the `input` field) as long as it works just fine with `mount`.

Reply:
I am not familiar with all the tests that we have, but the ones I've written/seen needed a `ThemeProvider`, yes.